quaddtype/numpy_quaddtype/src/scalar_ops.cpp

```diff
@@ -134,6 +134,9 @@ quad_richcompare(QuadPrecisionObject *self, PyObject *other, int cmp_op)
         Py_INCREF(other);
         other_quad = (QuadPrecisionObject *)other;
         if (other_quad->backend != backend) {
+            // we could allow, but this will be bad
+            // Two values that are different in quad precision, 
+            // might appear equal when converted to double.
```

Comment:
```suggestion
            // we could allow this, but it would be bad
            // since two values that are different in quad precision
            // might appear equal when converted to double.
```